### PR TITLE
Update AFJ core and configure multiple ledgers

### PR DESCRIPTION
**Pull request by amanji**
Signed-off-by: Akiff Manji <akiff.manji@gmail.com>

# Summary of Changes

* Upgrade AFJ core and react-native dependency to support multiple ledgers
* Adds Indy Pool configurations for multiple ledgers
* Initialize Agent with multiple Indy Pool configurations

# Related Issues

#100 

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [ ] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [ ] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [ ] Run prettier: `npm run style-format`
- [ ] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "pre-commit-lint": "npx lint-staged"
   },
   "dependencies": {
-    "@aries-framework/core": "0.1.0-alpha.276",
+    "@aries-framework/core": "^0.1.0-alpha.289",
     "@aries-framework/react-hooks": "^0.1.2",
     "@aries-framework/react-native": "0.1.0-alpha.276",
```
Comment (TimoGlastra) [inline package.json:20]: React native version should stay in sync with core version

**Review by TimoGlastra — CHANGES_REQUESTED** [review]
Nice work @amanji. One small comment

File: package.json (hunk 1 above)
Comment (amanji) [reply package.json:20]: Ah, thanks for the head up. I'll update that as well.

**Review by jleach — APPROVED** [review]
Looks good.

**Review by TimoGlastra — APPROVED** [review]
Nice!

**Pull request merged**